core/sds-aspect-model-java-generator/src/main/resources/java-static-class-property-lib.vm

```diff
@@ -364,16 +363,16 @@ Optional.of("$property.getPayloadName()")
     }
 #end
 
-#if( $util.hasContainerType( $property ) )
+#if( $util.hasContainerType( $property ) || ${propertyType.contains( "Either" )} )
```

Comment:
Because `hasContainerType` is used in other places in the template as well where Either should not trigger generation (i.e. creating `StaticContainerProperty`). Here it's only the decision to generate a `SuppressWarnings("unchecked")`.